Library/Format.swift

```diff
@@ -456,15 +475,27 @@ private struct NumberFormatterConfig {
     numberStyle: .decimal,
     roundingMode: .down,
     maximumFractionDigits: 0,
+    minimumFractionDigits: 0,
     generatesDecimalNumbers: false,
     locale: .current,
     currencySymbol: "$"
   )
 
+  fileprivate static let defaultDecimalCurrencyConfig = NumberFormatterConfig(
+    numberStyle: .decimal,
+    roundingMode: .down,
+    maximumFractionDigits: 2,
+    minimumFractionDigits: 2,
+    generatesDecimalNumbers: false,
+    locale: Locale(identifier: "en"), // Always format decimal currency amounts using En locale
```

Comment:
Can you elaborate on this?

Reply:
I've also noticed these `static` configurations are not sorted alphabetically so was just wondering if we could put `defaultDecimalCurrencyConfig` and `defaultCurrencyConfig` next to each other, because they both deal with currency?

Reply:
> Can you elaborate on this?

Since we're using this formatter to format currency amounts into `String`s for the API, we need to always use the `.en` locale because the API doesn't support accepting amounts that are formatted using a different locale. For example: `10.00` vs `10,00`.

Reply:
Could we possibly expand that comment and explain this is for the purposes of API/Graph?